pepper-apis/studybuilder-cli/studies/singular/patches/singular-new-events.conf

```diff
@@ -1,5 +1,34 @@
 {
   "events": [
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "MEDICAL_RECORD_RELEASE",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "COPY_ANSWER",
+        "copyConfigPairs": [
+          {
+            "source": {
+              "type": "ANSWER",
+              "questionStableId": "CONSENT_DEPENDENT_DATE_OF_BIRTH"
+            },
+            "target": {
+              "type": "ANSWER",
+              "questionStableId": "MRR_LEGAL_DOB"
+            }
+          }
+        ]
+      },
+      "preconditionExpr": """
+        !user.studies["singular"].forms["MEDICAL_RECORD_RELEASE"].instances[latest].hasPreviousInstance() &&
+        user.studies["singular"].forms["CONSENT_DEPENDENT"].hasInstance()
+      """,
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 3
+    },
```

Comment:
For adult dependent medical release there was one dob field in last page that we had not copied the dob.
Here it is now.
I considered doing this in separate patch, but this one patch is already trying to add a bunch of events so seemed easier to do it this way.